src/Task/Testing/Atoum.php

```diff
@@ -0,0 +1,170 @@
+<?php
+namespace Robo\Task\Testing;
+
+use Robo\Contract\CommandInterface;
+use Robo\Contract\PrintedInterface;
+use Robo\Task\BaseTask;
+
+/**
+ * Runs atoum tests
```

Comment:
Please put all documentation here, the same way it is done for other tasks like PHPUnit.
Real docs will be generated from it